kubernetes/katago-server/templates/django/migrate.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: django-migrate-job
+  labels:
+    {{- include "katago-server.labels" . | nindent 4 }}
+spec:
+  backoffLimit: 2
+  ttlSecondsAfterFinished: 900
```

Comment:
Why such an hight number of seconds ? Maybe it is ok but i don't understand the use case

Reply:
I want to be able to see the container for a while after the job inside it failed, to make it easier to inspect it and look at logs and so on. :)

Reply:
ok for me